### DISCUSSION TITLE
Checking in

### DISCUSSION THREAD

**Discussion by hoclun-rigsep** [re-rxjs/react-rxjs#230] [General]
Hello Gentlemen,

I want to confirm my continued interest in contributing here and—only if you think it will be a useful spur to you—perhaps you could update me on the dev priorities or any housekeeping, blocking tasks, blocking discussions, etc!

**Comment by josepot** [comment]
Hi @hoclun-rigsep ! Thanks a lot for "checking in" :slightly_smiling_face: Honestly, reading this message made me very happy. I **really** appreciate your willingness to keep helping us improve the library. I've been quite busy these last few days, but I will try to find the time during this next week to bring you up to date with what our next priorities with the library are.

I would really like to hear your thoughts on what's next. The thing is that the library is working well, it's being used in production and it's helping developers to ship better software with React. So, despite the fact that this is a very small and humble library, with no aspirations whatsoever to become mainstream: I'm very happy with what we've built here.

Anyways, one thing that we realized a few months ago is that as much as this library advertises itself as "React bindings for RxJS" that's not accurate... because this library is in fact the bindings for a particular way of handling state using Observables... And I've found myself wanting to use this way of managing state with Observables completely outside of the realm of React (like needing factory Observables that don't create memory leaks). So, @voliva and myself have started moving the core parts of the library [here](https://github.com/josepot/rxjs-state). We don't have a README yet, or docs or anything, gut we got 100% test coverage so far, and the idea is that in the future react-rxjs will become the bindings of that library (that for the lack of a better name, now we are referring to as "rxjs-state").

We think that if we explain how to manage state effectively with "rxjs-state" (using factory-observables, using stateful observables that don't create memory leaks, using `createSignal`, using `partitionByKey` and `combineKeys`, etc), then it would be easier to explain the mental models that we want to propose and that are completely independent from React... Anyways, I had a very long week and I'm pretty tired right now, but we will find the moment to better explain these things (cough @voliva cough, help me out here).

I'll promise that we will keep you posted :wink:
